manticore/ethereum.py

```diff
@@ -655,17 +662,57 @@ def make_symbolic_value(self):
         return ABI.SValue
 
     @staticmethod
-    def compile(source_code, contract_name=None):
+    def compile(source_code, contract_name=None, libraries=None):
         ''' Get initialization bytecode from a Solidity source code '''
-        name, source_code, bytecode, runtime, srcmap, srcmap_runtime, hashes, abi, warnings = ManticoreEVM._compile(source_code, contract_name)
+        name, source_code, bytecode, runtime, srcmap, srcmap_runtime, hashes, abi, warnings = ManticoreEVM._compile(source_code, contract_name, libraries)
         return bytecode
 
     @staticmethod
-    def _compile(source_code, contract_name):
+    def _link(bytecode, libraries=None):
```

Comment:
since this is a stateless staticmethod, can't we write some unit tests pretty easily?